openlibrary/core/bookshelves.py

```diff
@@ -6,6 +6,7 @@
 class Bookshelves(object):
 
     PRESET_BOOKSHELVES = {
+        'Remove from current list': -1,
```

Comment:
Since `-1` isn't a "real" shelf, we don't need this here; these values are only used for lower-level access (closer to DB). The `-1` only really applies at the UI/API level.

```suggestion
```

Reply:
Done